### ISSUE TITLE
`hack/generate-docs.sh`doesn't build on macOS 10.14 Mojave

### ISSUE THREAD

**Issue by patricks** [kubernetes/kubernetes#64913]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind feature

/sig Docs

**What happened**:
The `hack/generate-docs.sh`doesn't build on macOS 10.14 Mojave.

**What you expected to happen**:
The script should generate the docs.

**How to reproduce it (as minimally and precisely as possible)**:
run `hack/generate-docs.sh` on macOS 10.14 Mojave.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
Client Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.0-alpha.0.974+a7a081b5410fcf", GitCommit:"a7a081b5410fcf711b1f85169225974040051204", GitTreeState:"clean", BuildDate:"2018-06-08T08:57:07Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.4", GitCommit:"5ca598b4ba5abb89bb773071ce452e33fb66339d", GitTreeState:"clean", BuildDate:"2018-06-06T08:00:59Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

- OS (e.g. from /etc/os-release):
macOS 10.14 Mojava (Beta 1)

**Comment by dims** [comment]
@patricks whats the error you see?

**Comment by patricks** [comment]
This is the output of `generate-docs.sh`

```sh
+++ [0610 09:36:53] Building go targets for darwin/amd64:
    cmd/gendocs
    cmd/genkubedocs
    cmd/genman
    cmd/genyaml
# k8s.io/kubernetes/cmd/genkubedocs
/usr/local/Cellar/go/1.10.3/libexec/pkg/tool/darwin_amd64/link: /usr/local/Cellar/go/1.10.3/libexec/pkg/tool/darwin_amd64/link: combining dwarf failed: Unknown load command 0x32 (50)

# k8s.io/kubernetes/cmd/genman
/usr/local/Cellar/go/1.10.3/libexec/pkg/tool/darwin_amd64/link: /usr/local/Cellar/go/1.10.3/libexec/pkg/tool/darwin_amd64/link: combining dwarf failed: Unknown load command 0x32 (50)

!!! [0610 09:37:22] Call tree:
!!! [0610 09:37:22]  1: /Users/patrick/Developer/kubernetes/hack/lib/golang.sh:610 kube::golang::build_binaries_for_platform(...)
!!! [0610 09:37:22]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0610 09:37:22] Call tree:
!!! [0610 09:37:22]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0610 09:37:22] Call tree:
!!! [0610 09:37:22]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
make: *** [all] Error 1
!!! Error in ./hack/generate-docs.sh:36
  Error in ./hack/generate-docs.sh:36. 'make -C "${KUBE_ROOT}" WHAT="${BINS[*]}"' exited with status 2
Call stack:
  1: ./hack/generate-docs.sh:36 main(...)
Exiting with status 1
```

**Comment by jamsyoung** [comment]
I had this exact same problem. I ran across https://github.com/Homebrew/brew/pull/4334#issuecomment-396852061 which resolved the problem.

tl;dr:
```
installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
```

**Comment by patricks** [comment]
Interesting information, but it doesn't fix my problem. Did you do something else? Like rebuild go-lang?

**Comment by patricks** [comment]
I fixed it by uninstalling go@1.9 and installing it again.

**Comment by buianhthang** [comment]
@jamsyoung it works for me, thanks
